bin/grade_students.sh

```diff
@@ -837,44 +840,51 @@ while true; do
 	global_submission_time=""
 	global_gradeable_deadline=""
 	# call the helper function
+        echo "========================================================================"
 	grade_this_item $NEXT_DIRECTORY $NEXT_ITEM
 
+        ${SUBMITTY_INSTALL_DIR}/bin/grade_item.py ${NEXT_DIRECTORY} ${NEXT_TO_GRADE} ${ARGUMENT_UNTRUSTED_USER}
```

Comment:
this is temporary, while I'm testing.  To make sure I get the same answer both ways.  I will remove it soon (but not before the first PR is merged).  It doesn't cause a problem, just wastes time (not travis time, since travis doesn't use these scripts -- no end to end) & space (storing results of both gradings).